playbooks/adhoc/docker_storage_to_overlay2/docker_storage_to_overlay2.yml

```diff
@@ -0,0 +1,197 @@
+---
+# This playbook changes the Docker storage driver to overlay2.
+#
+# It requires support for the "overlay2.size" daemon storage option to
+# preserve the devicemapper disk quota ("dm.basesize").  This feature
+# is beyond Docker 1.12.  See:
+#
+#    https://github.com/moby/moby/pull/24771
+#
+#    and
+#
+#    https://github.com/moby/moby/pull/32977
+#
+# Note, this makes no attempt to migrate existing containers/images.
+# It momentarily stops docker.service and uses "atomic storage reset"
+# to wipe all local Docker storage.
+#
```

Comment:
I just noticed this comment. How about if this playbook was a role instead, so Ops could call it after we migrate containers onto new nodes? Then we could control how many hosts have their storage wiped at once, and make sure users' apps remain running on other nodes. (Otherwise this would be too much of an outage to do in production). @mwoodson @jupierce thoughts?

Reply:
@sdodson Do you guys have an existing framework we could plug destructive roles like this into? For example, it would drain 10% of nodes at a time, and then run whatever role you need. Like a yum update+reboot role, or this docker storage reset role.

(I have something that I use for that, but I'm supposed to check upstream before making custom playbooks).

Reply:
@sdodson unping. Turns out we have approval to just use my playbook as a wrapper for this.

Reply:
@dak1n1 Your playbook just wraps the group evaluation or have you moved this into a role?